### DISCUSSION TITLE
Install Homebrew through a .NET Core console app in macOS with CliWrap

### DISCUSSION THREAD

**Discussion by Sergtek** [Tyrrrz/CliWrap#128] [Q&A]
I am trying to install the Homebrew package manager through a .NET Core console application on a computer with macOS (Big Sur), as indicated on the Homebrew website, the following command must be entered in the mac terminal:

`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
`

If I enter it in the mac terminal directly, it works fine, but when I try to get the console application to execute the same command using CliWrap but I don't get any error, there is no response, just nothing happens. This is the code I am using:
```
   static void Main(string[] args)
    {
            var result = Cli.Wrap("/bin/bash -c \"$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)\"");
            result.ExecuteAsync();
            Console.WriteLine(result);
            Console.ReadLine();
      }
```
Any help would be appreciated.

**Comment by Tyrrrz** [comment]
There are multiple issues here:
1. You are not `await`ing the execution. What you call `result` is not the result but a command. See the readme for more info.
2. You are passing the whole command line as the target file path. Instead you should separate the target file path and command line arguments. Use `WithArguments()` to provide arguments (everything after `/bin/bash` in command line).
